src/pages/checkout/shipping-method.js

```diff
@@ -110,7 +150,7 @@ export class ShippingMethodPage extends Component {
 
   render () {
     const { cart } = this.props
-    const { checkout: { shippingAddress } } = this.props
+    const { thirdPartyPaymentMethods } = this.props
```

Comment:
can these two can be on the same line?

```suggestion
    const { cart, thirdPartyPaymentMethods } = this.props
```